firmware/targets/furi_hal_include/furi_hal_onewire.h

```diff
@@ -0,0 +1,79 @@
+/**
+ * @file furi_hal_onewire.h
+ * OneWire HAL API
+ */
+
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+#include "furi_hal_gpio.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * Sets the pin to normal mode (open collector), and sets it to float
+ * @param gpio
+ */
+void furi_hal_onewire_start_drive(const GpioPin* gpio);
+
+/**
+ * Sets the pin to normal mode (open collector), and clears pin EXTI interrupt.
+ * Used in EXTI interrupt context.
+ * @param gpio
+ */
+void furi_hal_onewire_start_drive_in_isr(const GpioPin* gpio);
+
+/**
+ * Sets the pin to interrupt mode (EXTI interrupt on rise or fall), and sets it to float
+ * @param gpio
+ */
+void furi_hal_onewire_start_interrupt(const GpioPin* gpio);
```

Comment:
Please comment which EXTI (which pin) can be used as an interrupt.
V9 table from https://docs.google.com/spreadsheets/d/1qAPFTwvMwbTp1hTAtZHlO6jpl9o-fXg49rPIQ1o7JiQ/edit?usp=sharing
(0, 1, 4, 5, 7, 8, 14).

Reply:
I'd love to add this, but I'm not sure what you mean. I'm not that initimate with electronics.

Are you saying all pins on A..H are interrupt-capable if they're on EXTI{0,1,4,5,7,8,14} and all others are not?

I.e.:
```c
/*
 * Sets the pin to interrupt mode (EXTI interrupt on rise or fall), and sets it to float
 * The following pins are valid:
 * gpio_infrared_rx, gpio_display_rst_n, gpio_ext_pc0, gpio_subghz_cs (EXTI0),
 * gpio_cc1101_g0, gpio_display_di, gpio_ext_pc1, gpio_spi_d_sck (EXTI1),
 * ... etc ...
```
?

I have no idea what I'm typing here, so I could be off by a mile :shrug:

Reply:
ping @DrZlo13 : any hint is appreciated :)